prov/usnic/src/usdf_cq.c

```diff
@@ -803,12 +782,13 @@ usdf_cq_make_soft(struct usdf_cq *cq)
 
 		/* fill in the soft part of union */
 		TAILQ_INIT(&cq->c.soft.cq_list);
-		cq->c.soft.cq_comps = calloc(cq->cq_attr.size, comp_size);
+		cq->c.soft.cq_comps = (struct usdf_cq_soft_entry *)calloc(cq->cq_attr.size,
```

Comment:
nit: omit the cast here, we're writing C not C++, so the `void *` will coerce automatically